docs/api/rpc-experimental.md

```diff
@@ -232,567 +169,3 @@ http post https://rpc.testnet.near.org jsonrpc=2.0 id=dontcare \
-                    "registers_memory_limit": 1073741824
-                },
-                "regular_op_cost": 3856371
-            }
-        },
-        "total_supply": "1009000000000000000000000000000000000",
-        "transaction_validity_period": 100,
-        "validators": [
-            {
-                "account_id": "node0",
-                "amount": "50000000000000000000000000000000",
-                "public_key": "ed25519:7PGseFbWxvYVgZ89K1uTJKYoKetWs7BJtbyXDzfbAcqX"
-            },
-            {
-                "account_id": "node1",
-                "amount": "50000000000000000000000000000000",
-                "public_key": "ed25519:6DSjZ8mvsRZDvFqFxo8tCKePG96omXW7eVYVSySmDk8e"
-            },
-            {
-                "account_id": "node2",
-                "amount": "50000000000000000000000000000000",
-                "public_key": "ed25519:GkDv7nSMS3xcqA45cpMvFmfV1o4fRF6zYo1JRR6mNqg5"
-            },
-            {
-                "account_id": "node3",
-                "amount": "50000000000000000000000000000000",
-                "public_key": "ed25519:ydgzeXHJ5Xyt7M1gXLxqLBW1Ejx6scNV5Nx2pxFM8su"
-            }
-        ]
-    }
-}
-```
-
-</p>
-</details>
-
-## Genesis Records
-
-- method: `EXPERIMENTAL_genesis_records`
-- params:
-  - (1) (optional) `pagination` object includes `offset` and `limit`
-
-If you pass empty parameters, you will get the default pagination, which is offset 0 and limit 100.
-
-```bash
-http post https://rpc.testnet.near.org jsonrpc=2.0 id=dontcare \
-          method=EXPERIMENTAL_genesis_records \
-          'params:={}'
-```
-
-Yet, you have control over the pagination:
-
-```bash
-http post https://rpc.testnet.near.org jsonrpc=2.0 id=dontcare \
-          method=EXPERIMENTAL_genesis_records \
-          'params:={"pagination": {"offset": 1, "limit": 2}}'
-```
-
-<details><summary>Sample response</summary>
-<p>
-
-```json
-{
-    "id": "dontcare",
-    "jsonrpc": "2.0",
-    "result": {
-        "pagination": {
-            "limit": 10,
-            "offset": 2
-        },
-        "records": [
-            {
-                "Account": {
-                    "account": {
-                        "amount": "1000000000000000000000000000000000",
-                        "code_hash": "11111111111111111111111111111111",
-                        "locked": "0",
-                        "storage_paid_at": 0,
-                        "storage_usage": 0
-                    },
-                    "account_id": "near"
-                }
-            },
-            {
-                "AccessKey": {
-                    "access_key": {
-                        "nonce": 0,
-                        "permission": "FullAccess"
-                    },
-                    "account_id": "near",
-                    "public_key": "ed25519:546XB2oHhj7PzUKHiH9Xve3Ze5q1JiW2WTh6abXFED3c"
-                }
-            }
-        ]
-    }
-}
-```
-
-</p>
-</details>
-
-## Transaction Status with Receipts
```

Comment:
why is this removed?

Reply:
This was supposed to be moved to `rpc.md` not removed entirely. My mistake, I will fix it :)